docs/specs/xref.yml

```diff
@@ -390,3 +390,135 @@ outputs:
   docs/1/b.json: |
     { "content": "<p>Link to <a href=\"../c\">title</a></p>" }
   docs/c.json:
+---
+# Markdown and SDP cyclic xref reference
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    ---
+    title: Title from yaml header a
+    uid: a
+    fullName: name for a
+    summary: Link to @b?displayProperty=summary
+    ---
+    Link to @b?displayProperty=summary
+  docs/b.json: | 
+    {
+      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+      "uid": "b",
+      "name": "name for b",
+      "description": "description for b",
+      "summary": "Link to @a?displayProperty=summary",
+    }
+outputs:
+  docs/b.json: |
+    { "content":{"name":"name for b","description":"description for b"} }
```

Comment:
>{ "content":{"name":"name for b","description":"description for b"} } [](start = 4, length = 69)

why b can be build successfully?

Reply:
Because while we build xref map, for `.md` YAML header we do not build all xref properties, just name and title.
https://github.com/dotnet/docfx/blob/258d0969ad3bd473b9536cc7ef73b07cf024bc14/src/docfx/build/xref/XrefMap.cs#L301